paper/paper.tex

```diff
@@ -50,11 +50,13 @@ \subsection{The enclose API}
 \end{figure}
 
 In \emph{\RE}, the \texttt{solver} is an instance of a struct that must be a subtype of \texttt{AbstractEnclosureAlgorithm}.
-If a user wants to add a new solver, they just have to add a new struct, say, \texttt{MyEnclosure} and implement the method \texttt{\_enclose}, which is automatically called by \texttt{enclose}.
+If a user wants to add a new solver, they just have to add a new struct, say, \texttt{MyEnclosure} and extend the method \texttt{enclose}, as the following code snippet demonstrates.
 %
 \begin{lstlisting}[language=Julia]
-_enclose(solver::MyEnclosure, f::Function,
-         D::Union{Interval, IntervalBox}; kwargs...)
+function enclose(f::Function, dom::Union{Interval,IntervalBox},
+                 solver::MyEnclosure; kwargs...)
+    # Write your algorithm here.
```

Comment:
We need another line break.

I also suggest to change the comment (to me an algorithm is something on the meta level, not an implementation).

```suggestion
function enclose(f::Function,
                 dom::Union{Interval,IntervalBox},
                 solver::MyEnclosure; kwargs...)
    # solver-specific implementation
```

Reply:
New wording sounds good